Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -80,6 +80,33 @@ To use OAuth 2.0 authorization follow the next steps:
 3. The grant type used to get an access token is `Resource owner password credentials`. See the [Snow documentation](https://docs.servicenow.com/bundle/xanadu-platform-security/page/administer/security/concept/c_OAuthApplications.html#d25788e201) for more information.
 
 
+
+
+#### Prerequisites in order to support JWT
+
+1. Create a Java Key Store and upload it to the instance (Accessing from the upper menu :All→System Definition→Certificates)
+(Private key will be used as integration parameter)
+2. Configure a JWT signing key (use keystore from above .Keep the Key Id -  it will be used as kid integration parameter) 
```

Comment:
```suggestion
1. Create a Java Key Store and upload it to the instance. (Accessing from the upper menu :**Al** > **System Definition** > **Certificates**.)
(Private key will be used as an integration parameter)
2. Configure a JWT signing key (Use the keystore from above. Keep the Key ID. It will be used as kid integration parameter) 
```
What is the Private key?